.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,53 @@
+name: E2E tests
```

Comment:
Can we call that `Starship tests`? We have already a job for e2e that will be deprecated eventually, but until then, the naming will be confusing.